files/chef-marketplace-gem/lib/marketplace/reckoner/config.rb

```diff
@@ -40,6 +41,24 @@ class Config
 
         configurable :product_code
       end
+
+      config_context :phone_home do
+        default :enabled, false
+        configurable :endpoint
```

Comment:
Yay, it is configurable.  We need to use this attribute though.  Perhaps just set a default here?